ci/script.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+source ./tempEnv.sh
+
+set -eEuo pipefail
+
+echo "=== Local variables ==="
+ENABLE_LOGS="${ENABLE_LOGS:-true}"
+ALLOW_AUDIT_FAILURES="${ALLOW_AUDIT_FAILURES:-false}"
+ALLOW_TEST_FAILURES="${ALLOW_TEST_FAILURES:-false}"
+PROD_RELEASE_BRANCH="${PROD_RELEASE_BRANCH:-master}"
+DEV_RELEASE_BRANCH="${DEV_RELEASE_BRANCH:-development}"
+DOCKER_IMAGE_NAME="${DOCKER_IMAGE_NAME:-zencash/blockscout-frontend}"
+
+# absolute path to project from relative location of this script
+workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
+echo "=== Workdir: ${workdir} ==="
+
+command -v docker &> /dev/null && have_docker="true" || have_docker="false"
+echo "=== Have docker: ${have_docker} ==="
+
+# Functions
+function fn_die() {
+  echo -e "$1" >&2
+  exit "${2:-1}"
+}
+
+echo "=== Checking if Docker is installed ==="
+if [ "${have_docker}" = "false" ]; then
+  fn_die "Docker is not installed. Exiting ..."
+fi
+
+echo "=== Checking if GITHUB_TOKEN is set ==="
+if [ -z "${GITHUB_TOKEN:-}" ]; then
+  fn_die "GITHUB_TOKEN variable is not set. Exiting ..."
+fi
+
+echo "=== Checking if DOCKER_WRITER_PASSWORD is set ==="
+if [ -z "${DOCKER_WRITER_PASSWORD:-}" ]; then
+  fn_die "DOCKER_WRITER_PASSWORD variable is not set. Exiting ..."
+fi
+
+echo "=== Checking if DOCKER_WRITER_USERNAME is set ==="
+if [ -z "${DOCKER_WRITER_USERNAME:-}" ]; then
+  fn_die "DOCKER_WRITER_USERNAME variable is not set. Exiting ..."
+fi
+
+docker_tag=""
+if [ "${IS_A_RELEASE}" = "true" ]; then
+  docker_tag="${TRAVIS_TAG}"
+elif [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+  if [ "${TRAVIS_BRANCH}" = "${PROD_RELEASE_BRANCH}" ]; then
+    docker_tag=latest
+  elif [ "${TRAVIS_BRANCH}" = "${DEV_RELEASE_BRANCH}" ]; then
+    docker_tag=dev
+  fi
+fi
```

Comment:
I think general approach can be if IS_A_RELEASE then build 2 tags: VERSION and latest unless we do not want ever refer to latest tag in our deployments and use VERSIONED tag all the time :shrug:

Reply:
For deployments, versioned tags should be used, including `-RC` tags/versions.
I believe that's what we currently do - @davehorizen can you confirm?

I agree w previous comments that this should be standardized

Reply:
We do use tags, these are set in .env file in the compose project

Reply:
This logic just generates dev tag on development branch merge, and latest tag on main branch merge. 
We probably also want to generate latest tag when creating the release tag though.

Reply:
@otoumas this is essentially what I was saying :) Typically you build VERSIONED tag + latest = the same build and image, but to keep in mind that with the current implementation and addition `latest` tag will be overwritten every time you build VERSIONED and/or PROD_BRANCH build. Just FYI